tests/manage/pv_services/test_cephfs_with_mds_network_failure.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import time
+from random import randint
+from ocs_ci.ocs import constants
+from threading import Thread
+from ocs_ci.ocs.cluster import get_mds_standby_replay_info
+from ocs_ci.helpers.helpers import disable_vm_network_for_duration
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.framework.pytest_customization.marks import vsphere_platform_required, tier1
+from ocs_ci.ocs.resources.pod import search_pattern_in_pod_logs
+
+log = logging.getLogger(__name__)
+
+
```

Comment:
polarion id, bugzilla markers are missing

Reply:
Added bugzilla marker, Which polarion ID should i mention?

Reply:
@paraggit Please create a test case in polarion and add that ID to the marker @pytest.mark.polarion_id("xxx")

Reply:
Done.